components/TestButton.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+import {Button, BUttonGroup} from 'react-bootstrap';
+
+import '../less/buttons.less';
+import '../less/colors.less';
+
+
+function TestButton(){
```

Comment:
For functional components that don't use any state, it's nice to write them using [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions):

```JavaScript
const TestButton = () => (
  <div>
    <button className="btn" type="button">Hello There!!</button>
  </div>
);
```